### ISSUE TITLE
Module not found: Can't resolve 'apolloClient' 

### ISSUE THREAD

**Issue by amka94amka** [erxes/erxes#578]
./src/modules/auth/containers/SignIn.js
Module not found: Can't resolve 'apolloClient' in '/home/user/Desktop/katka/erxes/src/modules/auth/containers'

Бүгдийг нь yarn install болон yarn dev хийгээд erxes folder дотроосоо yarn dev хийхээр ийм алдаа гараад байхын ? Help Me

**Comment by shinebayar-g** [comment]
Аль project вэ?  Энэ repo юу?

**Comment by amka94amka** [comment]
erxes-widget erxes-widget-api  erxes-api энэ 3ийг yarn dev хийсэн
erxes-ийг yarn install хийчээд yarn start хйихээр ийм алдаа заагаад байган

**Comment by shinebayar-g** [comment]
`.env` тохиргооны файлаа үүсгэж өгөөгүй харагдаж байна. `.env.sample` файлаасаа .env-файлаа үүсгээд тохируулаарай.

**Comment by amka94amka** [comment]
yes мэдээж бүх project дээрээ .env file үүсгээд тэрийгээ ямарч өөрчлөлт хийгээгүй ажлуулсан боловч ийм алдаа гарч байгаан

**Comment by shinebayar-g** [comment]
reproduce хийгдэхгүй л байна даа. git clone хийчээд .env-гээ үүсгээгүй байхад бол яг ийм алдаа гарч байна. Өөрөөр бол ийм алдаа гарахгүй.
Энэ дарааллаар үзээрэй:
```
git clone https://github.com/erxes/erxes.git
cp .env.sample .env
yarn install
yarn start
```
Зөвхөн erxes repo-г шинээр ингээд үздээ. Асуудалгүй ажиллах ёстой.
Мөн Node version, yarn version чинь ядаж latest LTS байх ёстой шүү. Node 6-гаас доошоо бараг ажиллахгүй байх.
